Platform/CoffeelakeBoardPkg/AcpiTables/AcpiTables.inf

```diff
@@ -42,7 +42,7 @@
   Fwst/Fwst.aslc
   Platform/CommonBoardPkg/AcpiTables/Tpm/Tpm2.aslc
   Platform/CommonBoardPkg/AcpiTables/Fpdt/Fpdt.aslc
-  Platform/CommonBoardPkg/AcpiTables/Tpm/TpmSsdt.asl
+  Ssdt/TpmSsdt.asl

```

Comment:
should we update common TpmSsdt.asl to add 0xB3 support so that each platform doesn't copy this file?

Reply:
UEFI payload TCG2SMM Implementation for  CFL/CML is different compare to other platforms. am not sure what is the impact on other platforms if we move it to common code.

Reply:
B3 port is used for data. it should have no impact if it is not used previously by other platforms.
Subash could comment on this one since he did this for all the platforms.

Reply:
Regarding IOB3 to common TpmSsdta.asl, we can do it. It would be don't care bit for EHL/TGL. We would need to write same value to B2 & B3 ports. 

It's better to adapt SMM dispatcher to TGC2SMM for CML/CFL platforms and use common asl file.
There are some issues i have seen earlier in CML for SMI generations after Windows boot and continued older TGC2SMM implementation.  Can we add a as seperate asl for CFL? Once SMI is fixed with SMM dispatcher we could move to common file for CML & CFL.

Reply:
sure.